### ISSUE TITLE
Improve instruction of instalation

### ISSUE THREAD

**Issue by gustawdaniel** [js-cookie/js-cookie#332]
On page

> https://github.com/carhartl/jquery-cookie

There is weak instruction of installation. Your package can be installed by `bower`. Please paste this command to README.md:

```
bower install --save jquery.cookie
```

So problems with including the script directly from GitHub should end.

**Comment by damusix** [comment]
Also some instructions on using`npm install js-cookie` would be nice.

**Comment by AustinPaquette** [comment]
Made a PR just so it was a single click for the maintainers.

Needed to take a break from work anyhow so this was a nice few moments of mindless work 😄

**Comment by FagnerMartinsBrack** [comment]
@damusix There's some docs [here](https://github.com/js-cookie/js-cookie/tree/8b70250875f7e07445b6a457f9c2474ead4cba44#module-loaders) (folow the link for ES6 module loaders).

@gustawdaniel jquery-cookie is not being maintained anymore

@AustinPaquette I'll take a look in your PR https://github.com/js-cookie/js-cookie/pull/333

**Comment by balexandre** [comment]
@damusix you can also check my example project where I load js-cookie into a nodeJs project (part of a SO question) for #326

**Comment by FagnerMartinsBrack** [comment]
[As discussed](https://github.com/js-cookie/js-cookie/pull/333#issuecomment-323485607) on #333. I've created a few issues to gather community feedback in regards what's the priority to document.

Closing for now, let's see how that goes.

Feel free to watch those issues. Once one of them is ready to be documented, I'll write a comment there and anyone will be able to create a PR documenting that tool.

Closing this one for now.
